### PR TITLE
layers: Trim out some more old MT renderpass junk

### PR DESCRIPTION
**Pull request by chrisforbes**
- RENDER_PASS_NODE::attachments[i].attachment was always i.
- This whole structure is noise; we can use the VkAttachmentDescription
  directly.

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -10247,36 +10236,36 @@ CmdBeginRenderPass(VkCommandBuffer commandBuffer, const VkRenderPassBeginInfo *p
         if (renderPass) {
             uint32_t clear_op_size = 0; // Make sure pClearValues is at least as large as last LOAD_OP_CLEAR
             cb_node->activeFramebuffer = pRenderPassBegin->framebuffer;
-            for (size_t i = 0; i < renderPass->attachments.size(); ++i) {
+            for (size_t i = 0; i < renderPass->pCreateInfo->attachmentCount; ++i) {
                 MT_FB_ATTACHMENT_INFO &fb_info = framebuffer->attachments[i];
-                VkFormat format = renderPass->pCreateInfo->pAttachments[renderPass->attachments[i].attachment].format;
```
Comment (tobine) [inline layers/core_validation.cpp]: Wow, this was slightly confusing. Good riddance.

File: layers/core_validation.cpp [hunk 2]
```diff
@@ -10026,17 +10026,6 @@ VKAPI_ATTR VkResult VKAPI_CALL CreateRenderPass(VkDevice device, const VkRenderP
         render_pass->hasSelfDependency = has_self_dependency;
         render_pass->subpassToNode = subpass_to_node;
 #if MTMERGESOURCE
```
Comment (tobine) [inline layers/core_validation.cpp]: Any need to keep this #define?

Comment (chrisforbes) [inline layers/core_validation.cpp]: No, I'll remove it.

**Comment by chrisforbes** [comment]
Pushed to master.

**Pull request closed without merging**